spec/Staking.md

```diff
@@ -216,143 +237,132 @@ state = {
-  balance: {
-    "0xAB..CD": 790,
-  },
-  delegation: {
-    "0x23..45": [ ["0x01..23", 500] ],
-    "0xAB..CD": [ ["0x01..23", 100], ["0x23..45", 110] ]
-  },
-  revocations: [
-    ["0xAB..CD", "0x01..23", 124000, 50],
-    ["0x23..45", "0x01..23", 124100, 500],
-  ]
-}
+## RedelegateCCS
 
-> after a block with timestamp 124000
+### Action
 
+  * Format: `[6, prev_delegatee, next_delegatee, quantity]`
+    - A `prev_delegatee` is an `AccountId`.
+    - A `next_delegatee` is an `AccountId`.
+    - `quantity` is a `u64` amount of CCS to redelegate to `next_delegatee` from `prev_delegatee`.
+
+   Executing this action is the same as executing the revoke action and the delegate action. The `quantity` should be less than the delegated quantity of `prev_delegatee` from the sender.
+
+### Example
+
+```
 state = {
   stakeholders: [ "0x23..45", "0xAB..CD" ],
   balance: {
-    "0xAB..CD": 840,
+    "0x23..45": 500,
+    "0xAB..CD": 900,
   },
   delegation: {
-    "0x23..45": [ ["0x01..23", 500] ],
-    "0xAB..CD": [ ["0x01..23", 50], ["0x23..45", 110] ]
-  },
-  revocations: [
-    ["0x23..45", "0x01..23", 124100, 500],
-  ]
+    "0xAB..CD": [ ["0x23..45", 100] ]
+  }
 }
 
-> after a block with timestamp 124100
+> "0xAB..CD" sends staking action [ 6, "0x23..45", "0x67..89", 10 ]
 
 state = {
   stakeholders: [ "0x23..45", "0xAB..CD" ],
   balance: {
     "0x23..45": 500,
-    "0xAB..CD": 840,
+    "0xAB..CD": 900,
   },
   delegation: {
-    "0xAB..CD": [ ["0x01..23", 50], ["0x23..45", 110] ]
+    "0xAB..CD": [ ["0x23..45", 90], ["0x67..89", 10] ]
   }
 }
 ```
 
+## SelfNominate
 
-## RedelegateCCS
+### Action
+
+  * Format: `[ 4, deposit, metadata ]`
+
+  See SELF_NOMINATE section in [Dynamic Validator](./Dynamic-Validator.md#SELF_NOMINATE)
+
+## ReportDoubleVote
 
 ### Action
 
-  * Format: `[6, prev_delegatee, next_delegatee, quantity]`
-    - A `prev_delegatee` is an `AccountId`.
-    - A `next_delegatee` is an `AccountId`.
-    - `quantity` is a `u64` amount of CCS to redelegate to `next_delegatee` from `prev_delegatee`.
+  * Format: `[ 5, metadata_seq, params, ...signatrues ]`
 
-   Executing this action is the same as executing the revoke action and the delegate action. The `quantity` should be less than the delegated quantity of `prev_delegatee` from the sender.
+  See REPORT_DOUBLE_VOTE section in [Dynamic Validator](./Dynamic-Validator.md#REPORT_DOUBLE_VOTE)
 
-### Example
+## ChangeParameters
+This transaction will change the common parameters when more than half of the stakeholders agree.
+It does not change other fields of the scheme file because there are fields related to the genesis block.
 
-```
+It also does not provide a voting feature.
+The vote initiator should collect the signatures through the off-chain.
+
+This transaction increases the `seq` of `Metadata` and changes the `params` of `Metadata`.
+The changed parameters are applied from the next block that the changing transaction is included.
```

Comment:
The changed parameters are applied from the next block that the changing transaction is included in.